### ISSUE TITLE
/new & /thanks - Trailhead - Epic

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#7143]
## Description

Epic for the creation (coding, QA, review, launch) and a/b testing, closing and poss

**Comment by hoosteeno** [comment]
@hoosteeno update Alex with AP review

**Comment by alexgibson** [comment]
Launching these pages is blocked on some key questions:

- Can we launch the redesigned pages before July 4, at our own discretion?
- If no, when do the pages need to launch?

**Comment by hoosteeno** [comment]
This page can land in prod anytime after 10am PDT 20190529.

**Comment by hoosteeno** [comment]
An experiment report is forthcoming. It finds no significant difference between v=a and v=b. We are green-lighted to replace the v=a treatment ("Control") with the v=b treatment ("Trailhead") to 100% of audiences after 10am PDT 20190529.

**Comment by alexgibson** [comment]
Pull request above to make the new design is now ready and awaiting code review. I've marked it as do-not-merge until the 29th.

**Comment by sghoseWI** [comment]
Report: https://docs.google.com/document/d/1un9HIrY_aFr_hstTSSzFU8R5p7YwgyP2ODbjQoqq9f0/edit

**Comment by ejregithub** [comment]
I'm going to close this issue as we've created a separate issue for the code review - #7217 and another for the launch on 5/29 -#7218
